### PR TITLE
Disabled functionality added

### LINKED ISSUE #17: selected disabled doesn't work
selected disabled doesn't work. I.e it doesn't disable the selection of disabled option

### PR DESCRIPTION
**Pull request by corientdev**
resolves #17

### CODE REVIEW

File: jquery.dropdown.js [hunk 1]
```diff
@@ -264,6 +264,9 @@
 
           // Close opened dropdowns
           $(".dropdownjs > ul > li").attr("tabindex", -1);
+          if (e.target.getAttribute('disabled')) {
+          return;
```
Comment (FezVrasta) [inline jquery.dropdown.js]: wrong indentation here, may you fix?

File: jquery.dropdown.js [hunk 2]
```diff
@@ -264,6 +264,9 @@
 
           // Close opened dropdowns
           $(".dropdownjs > ul > li").attr("tabindex", -1);
+          if (e.target.getAttribute('disabled')) {
```
Comment (FezVrasta) [inline jquery.dropdown.js]: I would use jQuery to read the attribute to keep consistency.

File: jquery.dropdown.js [hunk 3]
```diff
@@ -384,6 +391,11 @@
         $option.addClass("selected");
       }
 
+      if ($this.prop("disabled")) {
+        $option.attr("disabled", true);
+        $option.addClass("disabled");
```
Comment (FezVrasta) [inline jquery.dropdown.js]: Why do you need a class? It should be enough to use the `:disabled` CSS selector to style it I think?

File: jquery.dropdown.js (hunk 2 above)
Comment (corientdev) [inline jquery.dropdown.js]: Same thing is used at many places in script. I just copied it from above.

File: jquery.dropdown.js (hunk 3 above)
Comment (corientdev) [inline jquery.dropdown.js]: I just copied `selected` thing from your own code.

**Pull request merged**